_posts/2021-12-15-pulpcore-3.17-is-ga.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Pulpcore 3.17 is now Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# Pulpcore 3.17 release announcement
+
+We are happy to announce the release of Pulpcore 3.17!
+
+Let's take a look at the key features of this release.
+
+For a full list of bugfixes, as well as Plugin API enhancements and bugfixes, see the [changelog](https://docs.pulpproject.org/pulpcore/changes.html)
+
+### New compatibility process
+
+This is the first release where we will trial our  attempt to maintain plugin compatibility. For plugin maintainers, please read [increasing Pulpcore compatibility with plugin versions](https://pulpproject.org/2021/11/23/increasing-pulpcore-compatibility-with-plugin-versions/) for more information.
+
+## RBAC enhancements ([#9411](https://pulp.plan.io/issues/9411), [#9413](https://pulp.plan.io/issues/9413), [#9415](https://pulp.plan.io/issues/9415), [#9498](https://pulp.plan.io/issues/9498), [#9604](https://pulp.plan.io/issues/9604))
+
+RBAC functionality in Pulp now has a new Roles model. To introduce and support roles in RBAC, it was necessary to introduce some supporting functions as well as rework existing functionality to use roles.
+
+Here is a summary of the main introductions and changes to RBAC:
+
+* New endpoints added for user and group roles
+* Options to customize the roles in your viewset
+* New ``assign_role`` and ``remove_role`` functions
+* New views have been added to assign model-level and object-level roles to users and groups.
+* existing access policies on viewsets were also rewritten to use roles.
+
+RBAC's Content Guards have also been reworked to use roles, with the addition of the following new new endpoints to the RBAC content guard viewset:
+
+* ``add_roles``
+* ``remove_roles``
+* ``list_roles``
+* ``my_permissions``
+
+The same object-specific endpoints have been added to tasks and groups viewsets.
+
+If you ever want to revert to the default settings, a new `reset` endpoint has been added to provide for this.
```

Comment:
```suggestion
If you ever want to revert a modified access policy to the default settings, a new `reset` endpoint has been added to provide for this. This will clear the customized flag, too, so updates from the plugin will be received.
```
This was impossible before. Once you touched an AP, it was tainted forever and updated plugins would never update it again.